src/Util/Guid.php

```diff
@@ -33,7 +33,9 @@ public static function generateString(): string
 
     public static function generateAsHex(): string
     {
-        return self::generate()->getHex();
+        // string cast is a BC layer for gedmo/uuid v3
```

Comment:
don't you mean 
```suggestion
        // string cast is a BC layer for ramsey/uuid v3
```
?

Reply:
Yeah sorry xD.